packages/mui-joy/package.json

```diff
@@ -57,9 +57,9 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.17.2",
-    "@mui/base": "5.0.0-alpha.92",
-    "@mui/core-internal-download-tracker": "5.10.0",
```

Comment:
@michaldudak was it intentional that you didn't use "^" here?

Reply:
I don't think it matters in this case - we'll always release new versions of the tracker, Material UI, and Joy together so that they will be updated together. Having `^` wouldn't change much, unless I'm missing something.

Reply:
Let's use "^" to be consistent with the other packages.